lib/Drupal/Generic/Drupal7.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Generic\Drupal7.
+ */
+
+namespace Drupal\Generic;
+
+class Drupal7 {
+  public function __call($method, $args) {
+    if (function_exists('drupal_' . $method)) {
```

Comment:
Not too keen on that specialization.

I think

```
$this->drupal7->drupal_get_filename()
```

is better than

```
$this->drupal7->get_filename()
```

where you don't know which function it will call ...